lib/src/view/design_main_dna_sequence.dart

```diff
@@ -69,7 +73,14 @@ class DesignMainDNASequenceComponent extends UiComponent2<DesignMainDNASequenceP
         Domain next_dom = this.props.strand.substrands[i + 1];
         if (should_draw_domain(prev_dom, side_selected_helix_idxs, props.only_display_selected_helices) &&
             should_draw_domain(next_dom, side_selected_helix_idxs, props.only_display_selected_helices)) {
-          dna_sequence_elts.add(this._dna_sequence_on_loopout(loopout, prev_dom, next_dom));
+          if(props.show_loopout_length_main_view && (props.show_dna == false)){
```

Comment:
make sure that you are using the Dart auto-formatter. (In WebStorm it's Ctrl+Alt+L; in Visual Studio Code I think it might be Shift+Alt+F, but I'm not sure.) I can tell it's not being used because it would put a space between `if` and `(`, for instance, and between `)` and `{` at the end of the line.

It might not seem like a big deal, but I'm quite pedantic about having everything be formatted by the Dart formatter, so that the project looks consistent and will be easier to read by someone who, for example, comes from another Dart project.

(Of course we're using `snake_case` instead of `camelCase`, which breaks [recommended Dart convention](https://dart.dev/guides/language/effective-dart/style#do-name-other-identifiers-using-lowercamelcase), but I port so much code from the Python library, where `snake_case` is the convention, that I thought it's better to keep those the same.)